codewof/programming/content/en/sum-lucky-7/question.md

```diff
@@ -0,0 +1,8 @@
+# Sum lucky 7
+
+Create a function `sum_lucky_7(list)` that takes a list of integers and **prints** the sum of the elements in the list.
+Except the number 7 is a lucky number, so we double it before adding it to the sum, and the number immediately after a 7 in the list is also doubled (if it is not a 7).
```

Comment:
This is very confusing; like 7, 7, 8 (for example) could be:

14 + 14 + 8
14 + 7 + 16
14 + 7 + 8

Consider:
> However, the number 7 is a lucky number. All 7s should be doubled before adding to the sum, and any non-7 immediately after a 7 should also be doubled.

Or similar